src/related-items-and-comparisons/index.jsx

```diff
@@ -3,9 +3,9 @@ import CardItem from './CardItem.jsx';
 import RelatedItemsList from './RelatedItemsList.jsx';
 import './related-items-and-comparisons.scss';
 
-import { products, stylesForProducts, relatedProducts } from './sampleData.js';
+import { products } from './sampleData.js';
 
-const RelatedItemsAndComparisons = (props) => (
+const RelatedItemsAndComparisons = (/* { products } */) => (
```

Comment:
@jonathanjewett If at some point you want to start using API data, simply uncomment these props so you get:

```js
const RelatedItemsAndComparisons = ({ products }) => (
```

At that point you can also remove the ./sampleData.js import.

Reply:
Acknowledged